test/benchmarks.ts

```diff
@@ -12,13 +12,20 @@ process.on("unhandledRejection", (err) => {
 });
 
 type RepoInfo = {
-  name: string; // name of repo
-  location: string; // location to store repo within project
-  remoteLink: string; // where to clone repo from
-  targetGitRef: string; // what target should
-  filePatterns: Array<string>; // file patterns to lint in repo
-  browserslist?: Array<string>; // what browsers the repo should target
-  eslintOptions: ESLint.Options; // used in new ESLint(options)
+  /** name of repo */
+  name: string;
+  /** location to store repo within project */
```

Comment:
By convention, I think we should use inline comments here as opposed to multi-line comments